NEWS.md

```diff
@@ -117,6 +117,9 @@ Standard library changes
 * The `Pkg.Artifacts` module has been imported as a separate standard library.  It is still available as
   `Pkg.Artifacts`, however starting from Julia v1.6+, packages may import simply `Artifacts` without importing
   all of `Pkg` alongside. ([#37320])
+* `CartesianIndices` can now be constructed from three `CartesianIndex`es `I`, `S`, `J` with `I:S:J`.
```

Comment:
Before you say how to construct them, maybe it would be best to say that `CartesianIndices` now supports step different from 1?